Library/Formula/gws.rb

```diff
@@ -0,0 +1,22 @@
+class Gws < Formula
+  homepage "https://streakycobra.github.io/gws/"
+  url "https://github.com/StreakyCobra/gws/archive/0.1.7.tar.gz"
+  sha256 "36c25392f5cb93a39441a1973842249a62b605a2a7e369a040c78450683697d0"
+
+  depends_on "bash"
```

Comment:
@mikemcquaid yes, it depends on bash because it needs support for associative arrays 0d90928